frappe/public/scss/common/indicator.scss

```diff
@@ -3,14 +3,14 @@
 	color: var(--text-on-#{$color});
 	&::before,
 	&::after {
-		background: var(--text-on-#{$color});
+		background: var(--bg-#{$color});
```

Comment:
This will fix it for dark mode but now light mode will be broken
Also updating this CSS will affect the CSS of all indicator pill